### ISSUE TITLE
[Feature Request] Option to wait for AutoRetainer when time until completion is under X

### ISSUE THREAD

**Issue by CensoredFFXIV** [ffxivcode/AutoDuty#346]
Hey,

it would be nice if there was a option for waiting in between loops for AutoRetainer.
Because many times there is like around 5 minutes left on the wait and it then goes into the next 25 to 30min dungeon.

**Comment by Herculezz55** [comment]
yes this is awaiting IPCs from AR, i think its being added today? hopefully

**Comment by Herculezz55** [comment]
update to this the ipc's have been added and are just waiting to go from testing to release!!

**Comment by erdelf** [comment]
added via https://github.com/ffxivcode/AutoDuty/pull/1138
